### PR TITLE
[Trivial] remove unneeded spaces from init.cpp

### PR DESCRIPTION
**Pull request by Diapolo**

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -873,7 +873,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     nPruneTarget = (uint64_t) nSignedPruneTarget;
     if (nPruneTarget) {
         if (nPruneTarget < MIN_DISK_SPACE_FOR_BLOCK_FILES) {
-            return InitError(strprintf(_("Prune configured below the minimum of %d MiB.  Please use a higher number."), MIN_DISK_SPACE_FOR_BLOCK_FILES / 1024 / 1024));
+            return InitError(strprintf(_("Prune configured below the minimum of %d MiB. Please use a higher number."), MIN_DISK_SPACE_FOR_BLOCK_FILES / 1024 / 1024));
```
Comment (MarcoFalke) [inline src/init.cpp]: I did not touch this last time because the translation was already done. Am I wrong?

Comment (Diapolo) [inline src/init.cpp]: IMHO your changes with removing the "Error:" in front require a few re-translations, so that shouldn't matter.

Comment (MarcoFalke) [inline src/init.cpp]: Right, ACK on this one.

File: src/init.cpp [hunk 2]
```diff
@@ -1281,10 +1281,10 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
                     break;
                 }
 
-                // Check for changed -prune state.  What we are concerned about is a user who has pruned blocks
+                // Check for changed -prune state. What we are concerned about is a user who has pruned blocks
                 // in the past, but is now trying to run unpruned.
                 if (fHavePruned && !fPruneMode) {
-                    strLoadError = _("You need to rebuild the database using -reindex to go back to unpruned mode.  This will redownload the entire blockchain");
+                    strLoadError = _("You need to rebuild the database using -reindex to go back to unpruned mode. This will redownload the entire blockchain");
```
Comment (MarcoFalke) [inline src/init.cpp]: This would trigger new translations? So I'd NACK this one.

**Comment by MarcoFalke** [comment]
Do we care about white space in comments? @laanwj said this will break `git blame`.

**Comment by Diapolo** [comment]
Sometimes I'm asking myself, why it seems just my trivial pulls create controversial discussions ^^.
@MarcoFalke If this gets NACKed in general, fell free to pick it up.

**Comment by laanwj** [comment]
> Sometimes I'm asking myself, why it seems just my trivial pulls create controversial discussions ^^

Because, to be entirely honest, all this trivial fudging around with spaces tends to annoy me as long as there are still 341 actual issues to be solved.

File: src/init.cpp (hunk 1 above)
Comment (MarcoFalke) [inline src/init.cpp]: @Diapolo Maybe just get in this change? You can either force push it or wait until I submit my trivial branch (~every two weeks) with the trivial changes cherry-picked and bundled up.

**Comment by btcdrak** [comment]
I think what would help going forward is to have some sort of simple CS checker as part of the continuous integration like travis, then less CS issues would get into the codebase in the first place.

**Pull request closed without merging**